docs/deb-package-install.adoc

```diff
@@ -10,10 +10,10 @@ sudo apt install ./aktualizr.deb
 
 The debian package will install, enable, and start an `aktualizr` systemd service immediately after it's installed. However, there are some configuration steps that should be taken before the service starts. To use aktualizr with a server (i.e. https://github.com/advancedtelematic/ota-community-edition/[OTA Community Edition] or https://docs.atsgarage.com[HERE OTA Connect]), you will need to download the provisioning credentials file provided by the server and place it at `/var/sota/sota_provisioning_credentials.zip`.
 
-Additionally, if you wish to pass extra command line arguments to aktualizr, you can do so by creating `/usr/lib/sota/sota.env` to set the `AKTUALIZR_SECONDARY_CONFIG` environment variable. For example, to enable a secondary ECU with a config file at `/var/sota/sec_ecu.json`, the contents of the file would be:
+Additionally, if you wish to pass extra command line arguments to aktualizr, you can do so by creating `/usr/lib/sota/sota.env` to set the `AKTUALIZR_SECONDARY_CONFIG` environment variable. For example, to set up the log level to debug:
 
 ----
-AKTUALIZR_SECONDARY_CONFIG="--secondary-config /var/sota/sec_ecu.json"
+AKTUALIZR_SECONDARY_CONFIG="--loglevel 0"
```

Comment:
Ah, now I see. Can we change that variable safely? Or do we even care, since debian support is not actually essential anymore?

Besides, you removed `AKTUALIZR_SECONDARY_CONFIG` above from the `.service` file (probably rightfully so).